content/css/concepts/combinators/combinators.md

```diff
@@ -0,0 +1,74 @@
+---
+Title: 'Combinators'
+Description: 'Combinators are CSS selectors that are used to style elements that have a certain relationship with other selected elements.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Selectors'
+  - 'Elements'
+  - 'Style'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+---
+
+Combinators are CSS [selectors](https://www.codecademy.com/resources/docs/css/selectors) that are used to style elements that have a certain relationship with other selected elements.
+
+## Sytnax
+
+```pseudo
+selector1 selector2 ... selectorN {
+  /* declarations here */
+}
+
+selector1 > selector2 ... > selectorN {
+  /* declarations here */
+}
+
+selector1 + selector2 ... + selectorN {
+  /* declarations here */
+}
+
+selector1 ~ selector2 ... ~ selectorN {
+  /* declarations here */
+}
+```
+
+<!-- markdownlint-disable MD038 -->
+
+- The space selector (` `) matches all descendants of a specified element.
+<!-- markdownlint-enable MD038 -->
+- The child selector (`>`) matches all children of a specified element.
+- The adjacent sibling selector (`+`) matches all elements directly after the preceding element.
+- The general sibling selector (`~`) matches all elements that are next to the preceding element.
+
+## Example
+
+The example below shows how these combinators can be used:
```

Comment:
The rendered output image might be a little easier to follow if there was some separation between the examples, like a `<hr />` or just some extra spacing.